packages/uxpin-merge-cli/src/steps/serialization/vcs/repositories/fake/FakeRepositoryAdapter.ts

```diff
@@ -0,0 +1,26 @@
+import { MovedFilePathsMap } from '../../../DesignSystemSnapshot';
+import { AbstractRepositoryAdapter } from '../AbstractRepositoryAdapter';
+import { CommitMetadata, RepositoryAdapter } from '../RepositoryAdapter';
+import { getFakeCommit } from './util/getFakeCommit';
+
+export class FakeRepositoryAdapter extends AbstractRepositoryAdapter implements RepositoryAdapter {
```

Comment:
I had to add this because in npm integration we don't use version control and this was the fastest way to push library using push endpoint